### ISSUE TITLE
Consider Support for Operator, Chains, and Hub

### ISSUE THREAD

**Issue by danielhelfand** [danielhelfand/tekton-install#46]
There should be some research into whether supporting an install process through `tekton-install` for other Tekton components makes sense. Theses components include the following:
* [Operator](https://github.com/tektoncd/operator)
* [Chains](https://github.com/tektoncd/chains)
* [Hub](https://github.com/tektoncd/hub)

Closing this issue would mean providing an explanation why or why not to consider supporting installing the components above through `tekton-install`.

**Comment by vdemeester** [comment]
@danielhelfand note that the goal of the operator is to manage the component lifecycle. So if `tekton-install` supports the operator, it could rely solely on it for the rest of the components (at least the one the operator manages). We need to discuss so UX around this but 👍🏼

**Comment by danielhelfand** [comment]
Yeah, completely agree. I always saw making `tekton-install` more for development work around Tekton or for showcasing Tekton. Mainly setting up everything on a cluster really quick with slightly better UX than kubectl.

My hope is that most would use the operator for managing all aspects of Tekton components, so as you suggest, this could become a programmatic way of installing the operator, which could be really interesting. Then also using `tekton-install` to call on the operator to manage components. Open to any ideas around it.
